project-backend/Controllers/JobController.cs

```diff
@@ -128,6 +129,24 @@ public IActionResult GetJobDetails([FromQuery] GetJobDetailsQueryObject job)
             }
 
         }
+
+        [HttpPost]
+        [Authorize]
+        [Route("Bids")]
+        [ProducesResponseType(typeof(GetJobBidsResponseObject), StatusCodes.Status200OK)]
+        public IActionResult GetJobBids([FromBody] GetJobBidsQueryObject query)
```

Comment:
Why is this a post request?

Reply:
Also, didn't realize it but I guess this also should be in the bid controller. Just like add bid is in the bid controller, this would also be more sensible there.

Reply:
I would really disagree with that honestly. It doesn't even sound right from a route point of view. It's a post because we have pagination so it's easier to sue body.

Reply:
But then why did we agree on the fact that the equivalent of this function in the providers belongs to the bid provider?

Reply:
And true, we have pagination but there are no lists/complex data, so query params might work just as well here.

Reply:
The bid provider is closer to the storage representation from my point of view. As I said even there, we are interested in a part of a job, but the provider is internal so I agreed that we can move it. The controller shapes the API so at least my point of view, is that it makes a lot of sense that a job related part is in the job part of the api.

Reply:
Yes, but were you to write frontend code, where would you look first for a function that fetches bids?

Reply:
If it fetches bids, in the bid controller. If it fetches the bids of a certain job, in the job controller.